backend/src/main/java/com/redhat/cloud/notifications/recipients/itservice/pojo/request/AccountRelationship.java

```diff
@@ -0,0 +1,37 @@
+package com.redhat.cloud.notifications.recipients.itservice.pojo.request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
General comment: since `@JsonInclude` only applies to serialization, it can be removed from all classes in this PR as we only do deserialization.

Reply:
removed. thx